src/routes/user.ts

```diff
@@ -6,6 +6,8 @@ const userRouter: Router = Router();
 
 userRouter.get("/", userValidator("GET /users"), userController.index);
 
+userRouter.get("/resetLastSubmissions", userController.resetLastSubmissions);
```

Comment:
This should be a POST or PATCH or PUT request imo, not a GET

Reply:
This should be done for clarity/logical reasons, but also since our gateway blocks things like POSTS unless you're admin I think, so it's temporarily secure until we fix auth

Reply:
our gateway doesn't block POST for users (they wouldn't be able to login or signup if it was blocked)

Reply:
In fact, POST routes for users are the only ones that can be accessed without tokens

Reply:
Hmm okay. Well either way it doesn't make sense to have this as a GET, from a logical perspective.

Reply:
I'll make it a patch. Had it as a get since the route doesn't take in a payload and I didn't want to specify Content-Length 0 every time with requests :)

Reply:
@ValerianClerc fixed 👍